client/src/components/Authentication/Login.tsx

```diff
@@ -4,15 +4,14 @@ import { Link, useNavigate } from 'react-router-dom';
 import { Card, Button, Form, Alert } from 'react-bootstrap';
 
 // Local imports
-import { useAuth } from '../../context/AuthContext';
-// import { LoginUser } from '../../interfaces/Authentication/Login.interface';
+import { useAuth} from '../../context/AuthContext';
```

Comment:
It detects this as 2 line changes instead of one cause of the spacing around 'useAuth'. Could you please run the linter and see if it fixes that (for this and all files)?

Reply:
done